app/stac_api/models.py

```diff
@@ -633,17 +594,6 @@ def __str__(self):
     # alter save-function, so that the corresponding collection of the parent item of the asset
     # is saved, too.
     def save(self, *args, **kwargs):  # pylint: disable=signature-differs
-        self.item.collection.update_geoadmin_variants(
-            self.geoadmin_variant, self.proj_epsg, self.eo_gsd
-        )
-        if self.eo_gsd is not None:
-            self.item.update_properties_eo_gsd(self)
+        self.item.collection.update_summaries(self.geoadmin_variant, self.proj_epsg, self.eo_gsd)
 
         super().save(*args, **kwargs)
-
-    def delete(self, *args, **kwargs):  # pylint: disable=signature-differs
-        # It is important to use `*args, **kwargs` in signature because django might add dynamically
-        # parameters
-        if self.eo_gsd is not None:
-            self.item.update_properties_eo_gsd(self, deleted=True)
-        super().delete(*args, **kwargs)
```

Comment:
The delete case for the eo:gsd in collection.summaries was not implemented. As it has an impact on Performance, I would not do it in this PR, but I would suggest to leave the delete method as follow:
```python
def delete(self, *args, **kwargs):  # pylint: disable=signature-differs
    # pylint: disable=fixme
    # TODO implement delete case for updating collection.summaries [BGDIINF_SB-1462]
    super().delete(*args, **kwargs)
```
I've created  a ticket, see https://jira.swisstopo.ch/browse/BGDIINF_SB-1462

Reply:
If I remember correctly, I implemented it during my work for 1410 and can revert deletion from the branch I pushed for that case.
I'll check that tomorrow.

Reply:
I'll do that in https://jira.swisstopo.ch/browse/BGDIINF_SB-1462